app/styles/_state.scss

```diff
@@ -29,6 +29,7 @@
   /*Pull the message up to bring it closer to the header*/
   &.nudge {
     top: -10px;
+    padding: 10px;
```

Comment:
The padding was added to better reflect what @ryanfeeley had in his [mockups](https://github.com/mozilla/fxa-content-server/issues/3399). Without the padding it looks like this:

<img width="461" alt="screen shot 2016-02-08 at 10 11 20 am" src="https://cloud.githubusercontent.com/assets/1295288/12889331/6085e80c-ce4c-11e5-98d2-b354b547572e.png">

Also, I think I need to update the padding to only be applied for amo.